docs/_includes/button.html

```diff
@@ -388,4 +374,158 @@ <h3 id="button-wide">Wide Button</h3>
+                        {% include symbol.html name="dropdown" %}
+                    </svg>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button class="btn" aria-expanded="false" type="button">
+    <span class="btn__cell">
+        <span class="btn__text">Button</span>
+        <svg class="icon icon--dropdown" focusable="false" height="10" width="14" aria-hidden="true">
+            <use xlink:href="#icon-dropdown"></use>
+        </svg>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <p>Use the <span class="highlight">borderless</span> modifier to remove the border from the expandable button. Remember, that without a border, the button always requires an alternative means of "click" affordance (such as an icon here).</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button class="btn btn--borderless" type="button">
+                <span class="btn__cell">
+                    <span class="btn__text">Button</span>
+                    <svg class="icon icon--dropdown" focusable="false" height="10" width="14" aria-hidden="true">
+                        {% include symbol.html name="dropdown" %}
+                    </svg>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button class="btn btn--borderless" type="button">
+    <span class="btn__cell">
+        <span class="btn__text">Button</span>
+        <svg class="icon icon--dropdown" focusable="false" height="10" width="14" aria-hidden="true">
+            <use xlink:href="#icon-dropdown"></use>
+        </svg>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <p>Use the <span class="highlight">form</span> modifier to create an expandable button style that matches the look of form controls (e.g. textbox, select).</p>
+    <div class="demo">
+        <div class="demo__inner">
+            <button class="btn btn--form" aria-expanded="false" type="button">
+                <span class="btn__cell">
+                    <span class="btn__text">Button</span>
+                    <svg class="icon icon--dropdown" focusable="false" height="10" width="14" aria-hidden="true">
+                        {% include symbol.html name="dropdown" %}
+                    </svg>
+                </span>
+            </button>
+        </div>
+    </div>
+
+    {% highlight html %}
+<button class="btn btn--form" aria-expanded="false" type="button">
+    <span class="btn__cell">
+        <span class="btn__text">Button</span>
+        <svg class="icon icon--dropdown" focusable="false" height="10" width="14" aria-hidden="true">
+            <use xlink:href="#icon-dropdown"></use>
+        </svg>
+    </span>
+</button>
+    {% endhighlight %}
+
+    <p>Use the <span class="highlight">btn--slim</span> modifier for a slimmer expand button when displaying an icon only.</p>
```

Comment:
I'm thinking about this:

Does it make sense that this is actually part of icon-button? Technically this is only an icon in a button. It seems like we're coming back to mixing this. 
What do you think about having the expanded code for icon-button and have that as the "slim" version? It might be a bit more code but it could simplify the selectors and extra options.

Reply:
Yeah, I can look at doing that. It does seem to have more in common visually though with "Button" than "Icon Button". None of the icon buttons have the squared "form" border style.

Reply:
I decided to just keep it how it is for now.